metricbeat/docs/fields.asciidoc

```diff
@@ -2402,111 +2402,121 @@ Total space (used plus free).
 
 
 [float]
-== memory Fields
+== load Fields
 
-`memory` contains local memory stats.
+Load averages.
 
 
 
 [float]
-=== system.memory.total
+=== system.load.1
 
-type: long
+type: half_float
 
-Total memory.
+Load average for the last minute.
 
 
 [float]
-=== system.memory.used.bytes
+=== system.load.5
 
-type: long
+type: half_float
 
-Used memory.
+Load average for the last 5 minutes.
 
 
 [float]
-=== system.memory.free
+=== system.load.15
 
-type: long
+type: half_float
 
-Available memory.
+Load average for the last 15 minutes.
 
 
 [float]
-=== system.memory.used.pct
+== memory Fields
 
-type: half_float
+`memory` contains local memory stats.
 
-The percentage of used memory.
 
 
 [float]
-== actual Fields
+=== system.memory.total
 
-Actual memory fields.
+type: long
 
+Total memory.
 
 
 [float]
-=== system.memory.actual.used.bytes
+=== system.memory.used.bytes
 
 type: long
 
-Actual used memory. This value is the "used" memory minus the memory used for disk caches and buffers. Available only on Unix.
+Used memory.
+
+
+[float]
+=== system.memory.used.pct
+
+type: half_float
+
+The percentage of used memory.
 
 
 [float]
-=== system.memory.actual.free
+=== system.memory.free
 
 type: long
 
-Actual available memory. This value is the "free" memory plus the memory used for disk caches and buffers. Available only on Unix.
+Free memory. For a human readable value use Available.
 
 
 [float]
-=== system.memory.actual.used.pct
+=== system.memory.available
 
-type: half_float
+type: long
 
-The percentage of actual used memory.
+Available memory is what it matters. It is calculated based on the OS. On Linux it is a sum of free memory and caches and buffers. On OSX consists of free and inactive memory.
 
 
 [float]
-== swap Fields
+=== system.memory.active
```

Comment:
I would like to have a more detailed description for some of these fields and possibly how the data is collected (e.g. read from /proc/meminfo or same values reported by `vmstat -a`).

> Active — The total amount of buffer or page cache memory, in kilobytes, that is in active use. This is memory that has been recently used and is usually not reclaimed for other purposes.

> Inactive — The total amount of buffer or page cache memory, in kilobytes, that are free and available. This is memory that has not been recently used and can be reclaimed for other purposes.